devito/arch/compiler.py

```diff
@@ -588,15 +600,15 @@ def __lookup_cmds__(self):
 
 class PGICompiler(Compiler):
 
-    _cpp = True
+    default_cpp = True
 
     def __init_finalize__(self, **kwargs):
 
-        self.cflags.remove('-std=c99')
+        self.cflags.remove(f'-std={self.std}')
         self.cflags.remove('-O3')
         self.cflags.remove('-Wall')
 
-        self.cflags.append('-std=c++11')
+        self.cflags.append(f'-std={self.std}')
```

Comment:
So heree you first remove and then append the same flag?
This will give different flags than before AFAIU

Reply:
No this removes the C standard (c99) and adds the c++ standard (c++11)

Reply:
ok, now I see the property....
not sure I like it much though

```
    @property
    def std(self):
        return self._cxxstd if self._cpp else self._cstd
```